.github/workflows/stale-issues.yml

```
@@ -22,13 +22,13 @@ jobs:
       - uses: actions/stale@v9
         with:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
-          stale-issue-message: 'Thanks for sharing your feedback. We apologize for our delayed response. Due to the sheer volume of feedback in our repo, we have decided to prioritize the newest issues and are automatically closing our oldest issues with an Issues bot. If you believe your feedback is still actionable, then either respond directly in this issue or open a new documentation issue and we’ll review. If no activity occurs in the next 14 days, this issue will be closed.'
+          stale-issue-message: 'Thanks for sharing your feedback. We apologize for our delayed response. Throughout 2024 we are phasing out Git issues as the feedback mechanism on content and replacing it with a new feedback system. If you believe your feedback needs to be reviewed prior to closing, then respond directly in this issue and we’ll take a look at it. If no activity occurs in the next 7 days, this issue will be closed.'
```

How can we avoid confusion with our product still indeed using GitHub for issues

```suggestion
          stale-issue-message: 'Thanks for sharing your feedback. We apologize for our delayed response. Throughout 2024 we are phasing out GitHub issues as the feedback mechanism on documentation content and replacing it with a new feedback system. If you believe your feedback needs to be reviewed prior to closing, then respond directly in this issue and we’ll take a look at it. If no activity occurs in the next 7 days, this issue will be closed.'
```

---

Yes, it's good to differentiate.